modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -71,6 +71,14 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following bug fixes:
 
+=== Resolving formatting issues with `contenteditable= "false"` contexts and the formatter API
+//#TINY-9678
+
+In previous versions of {productname}, formatting issues caused incorrect application or removal of formats in relation to the closest `contenteditable= "true"` element when the selection was within a `contenteditable= "false"` context.
+
+This issue also affected the formatters API as well, as its `canApply` function failed to return `false` when the selection was in a `contenteditable= "false"` context.
```

Comment:
Should it be`formatters API` or `formatter API`? It is different in the heading. 
Also remove extra space after `contenteditable`